### DISCUSSION TITLE
Caused by: jakarta.faces.FacesException: Error loading CSS, cannot find "theme.css" resource of "primefaces-poseidon-" library

### DISCUSSION THREAD

**Discussion by amran-bd** [primefaces/community#1043] [PrimeFaces Templates] [answered]
Hello Team, 
 
Could u  please help me to resolve this issue? 

I had a project that I used JSF 4, Jakarta 10, wildfly 30, Poseidon theme 6.0.0., and Java 21 after running successfully I found this issue in the browser.

**Comment by buraksaglam089** [comment, marked as answer]
Hello, 

Did you compile the sass with command before you run the project?

Regards.

**Reply by amran-bd** [reply]
Hi Burak,

Yes. After compile sass problem was resolved. 

Thanks.
